VERSION

```diff
@@ -1 +1 @@
-0.1.12
+0.1.14
```

Comment:
@dgursoy Not sure if this change was overlooked. Please make sure this is the version number tomopy should currently be on.